### PR TITLE
Checks

### PR DESCRIPTION
**Pull request by Sybrendj2001**
In admin i changed addtime and edittime. Also i fixed the namespace for customer and added duration and genre for the movies

### CODE REVIEW

File: CinemaConsole/Pages/Admin/Admin.cs [hunk 1]
```diff
@@ -605,7 +604,15 @@ private static void Add()
...
 
-                CD.InsertMovie(name, releaseDate, age, sum, actors);
+                Console.WriteLine("\nPlease enter the movie duration in minutes");
+                int duration = Convert.ToInt32(Console.ReadLine());
+
+                Console.WriteLine("\nPlease enter the movie genre");
+                string genre = Console.ReadLine();
+
+                CD.InsertMovie(name, releaseDate, age, sum, actors, duration, genre);
+
```
Comment (timfeskens) [inline CinemaConsole/Pages/Admin/Admin.cs:619]: remove whitespace

File: CinemaConsole/Pages/Admin/Admin.cs [hunk 2]
```diff
@@ -827,38 +837,46 @@ public static void addTime(string title)
...
+                                else
                                 {
-                                    y = false;
+                                    Console.WriteLine("This movie already exists at this time in this theather");
+                                    break;
                                 }
                             }
                             catch
                             {
+                                //TODO: Add add catch
```
Comment (timfeskens) [inline CinemaConsole/Pages/Admin/Admin.cs]: add a catch

File: CinemaConsole/Pages/Admin/Admin.cs [hunk 3]
```diff
@@ -827,38 +837,46 @@ public static void addTime(string title)
...
                         {
-                            Customer.Customer.showTime(AD.GetMovieID(title).ToString());
+                            DateTimeHall datetimehall1 = new DateTimeHall(DT, hall, title);
+                            Console.Clear();
+                        }
 
+                        while (!checksdone)
+                        {
+                            Customer.showTime(AD.GetMovieID(title).ToString());
+
```
Comment (timfeskens) [inline CinemaConsole/Pages/Admin/Admin.cs:900]: remove whiteline

**Review by timfeskens — CHANGES_REQUESTED** [review]
>:(

**Review by jjtc19 — APPROVED** [review]

**Comment by timfeskens** [comment]
closed instead of merged

**Review by timfeskens — APPROVED** [review]

**Pull request merged**